### ISSUE TITLE
Which accuracy do I need to reach for a decent agent?

### ISSUE THREAD

**Issue by PeterLeon** [Sentdex/pygta5#73]
I ve trained my model by using 200gb of driving data and inception_v3 model on same street with same weather conditions and 8.000 epochs with 300kb batches per epoch and learning rate of 0.000000001.
Accuracy is somewhat about 0.2(smoothed).
But when I test it out it keeps pressing WD with a verage plus of 0.3 of probability.

My question, how much data and which acc. do I need for a decent agent?

**Tensorboard**:
http://honesthome.ddns.net/s.png

**Example Output [W, WA, WD, S, SA, SD, A, D]:**
 [ 0.1167456   0.16700381  0.27807635  0.05266069  0.08322734  0.1091948
  0.11513347  0.07795788]
0.278076350689
forward_right
[ 0.11684461  0.16718809  0.27833453  0.0525772   0.08328342  0.1088759
  0.11493194  0.07796425]
0.278334528208
forward_right
[ 0.11666095  0.16800123  0.27791589  0.05253068  0.08310306  0.10876592
  0.11503578  0.07798637]
0.277915894985
forward_right
[ 0.116884    0.16714898  0.2784971   0.05255117  0.08334375  0.10875834
  0.11481125  0.0780055 ]
0.278497099876
forward_right
[ 0.11672532  0.16719386  0.27897081  0.05252673  0.08339871  0.10860606
  0.11471127  0.07786718]
0.278970807791
forward_right
[ 0.11685108  0.16760175  0.27800107  0.05256343  0.08338011  0.10876746
  0.11482292  0.07801216]

**Comment by germanmarky** [comment]
0.2 accuracy (20%) is quite low, and with 8 different predictions, even a random prediction is 0.125 (12.5%). If you trained your model at a learning rate of 1e-9, that is VERY low, you shouldn't start it any lower than 1e-5, I would use 1e-3 for one epoch, see how the accruacy/loss were progressing, and adjust down accordingly for later epochs.
